packages/notifi-core/lib/models/ConversationMessages.ts

```diff
@@ -6,16 +8,9 @@ export type ConversationMessages = Readonly<{
   };
 }>;
 
-export type ConversationMessagesEntry = Readonly<{
-  __typename?: 'ConversationMessage';
-  conversationParticipant: Participant;
-  id: string;
-  message: string;
-  userId: string;
-  createdDate: string;
-  updatedDate: string;
-  conversationId: string;
-}>;
+export type ConversationMessagesEntry = NonNullable<
```

Comment:
question! do we need the nonNullable around the nonNullable?

Reply:
Nice catch!! it is the same. Thanks for pointing out. resolved.